general/forall.hpp

```diff
@@ -61,38 +61,33 @@ const int MAX_Q1D = 14;
 // The MFEM_FORALL wrapper
 #define MFEM_FORALL(i,N,...)                             \
    ForallWrap<1>(true,N,                                 \
-                 [=] MFEM_DEVICE (int i) {__VA_ARGS__},  \
-                 [&] MFEM_LAMBDA (int i) {__VA_ARGS__})
+                 [=] MFEM_DEVICE (int i) {__VA_ARGS__})
```

Comment:
Is there a reason this isn't `MFEM_HOST_DEVICE` ?

Reply:
You're right, it should be `MFEM_HOST_DEVICE`: I'll update it, thank you @tomstitt